### DISCUSSION TITLE
blog/20240110/

### DISCUSSION THREAD

**Discussion by giscus-x** [xaoxuu/blog-comments#41] [Announcements]
# blog/20240110/

近日发现一个用 NSNotificationCenter 发通知时触发的 EXC_BAD_ACCESS 崩溃，表现形式比较奇怪，特此记录一下。

https://xaoxuu.com/blog/20240110/

**Comment by penndu** [comment]
看不懂但感觉很厉害~

**Reply by xaoxuu** [reply]
杜老师研究一下 😀

**Reply by penndu** [reply]
你太高看我了~

**Comment by penndu** [comment]
主题更新了吗？感觉和之前的变化很大~

**Reply by xaoxuu** [reply]
嗯，侧边栏可以设置背景了

**Comment by kuhehe** [comment]
大佬，你最后两个个红色代码框连在一起，中间用虚线隔开，是怎么弄的呀？谢谢

**Reply by xaoxuu** [reply]
child:codeblock
里面直接放两个代码块就行
